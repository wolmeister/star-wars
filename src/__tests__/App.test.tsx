import React from 'react';
import { render } from '@testing-library/react';

import App from '~/App';

describe('App', () => {
  test('renders correctly', () => {
    const app = render(<App />);
    expect(app).toMatchSnapshot();
  });
});
