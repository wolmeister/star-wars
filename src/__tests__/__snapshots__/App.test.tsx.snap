// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-hMqMXs eAZbBk"
      >
        <div
          class="sc-kEYyzF bWrbfR"
        >
          <div
            class="sc-kkGfuU eJiDXk"
          >
            <img
              class="sc-iAyFgw jrQOOC"
              src="test-file-stub"
            />
            <div
              class="sc-hSdWYo vEDBE"
              style="height: 0px;"
            />
            <div
              class="sc-eHgmQL jmEFwa"
            >
              <div
                class="sc-bdVaJa fYJyil"
              >
                <div
                  class="sc-bxivhb bojDWV"
                  style="transform: none;"
                >
                  <div
                    class="sc-bwzfXH sc-htpNat iqsgvA"
                    color="#C4C4C4"
                    style="width: 0px;"
                  />
                  <div
                    class="sc-bwzfXH bYnwOI"
                    color="#FFE81F"
                    style="width: 0px;"
                  />
                </div>
                <div
                  class="sc-gzVnrw ibZhkv"
                  style="opacity: 0;"
                >
                  <button
                    class="sc-htoDjs jnpWqT"
                    style="opacity: 0;"
                  >
                    <img
                      alt="previous-items"
                      src="test-file-stub"
                    />
                  </button>
                  <button
                    class="sc-htoDjs ijxUTm"
                    style="opacity: 0;"
                  >
                    <img
                      alt="next-items"
                      src="test-file-stub"
                    />
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-hMqMXs eAZbBk"
    >
      <div
        class="sc-kEYyzF bWrbfR"
      >
        <div
          class="sc-kkGfuU eJiDXk"
        >
          <img
            class="sc-iAyFgw jrQOOC"
            src="test-file-stub"
          />
          <div
            class="sc-hSdWYo vEDBE"
            style="height: 0px;"
          />
          <div
            class="sc-eHgmQL jmEFwa"
          >
            <div
              class="sc-bdVaJa fYJyil"
            >
              <div
                class="sc-bxivhb bojDWV"
                style="transform: none;"
              >
                <div
                  class="sc-bwzfXH sc-htpNat iqsgvA"
                  color="#C4C4C4"
                  style="width: 0px;"
                />
                <div
                  class="sc-bwzfXH bYnwOI"
                  color="#FFE81F"
                  style="width: 0px;"
                />
              </div>
              <div
                class="sc-gzVnrw ibZhkv"
                style="opacity: 0;"
              >
                <button
                  class="sc-htoDjs jnpWqT"
                  style="opacity: 0;"
                >
                  <img
                    alt="previous-items"
                    src="test-file-stub"
                  />
                </button>
                <button
                  class="sc-htoDjs ijxUTm"
                  style="opacity: 0;"
                >
                  <img
                    alt="next-items"
                    src="test-file-stub"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
